plugins/generator-1.15.2/forge-1.15.2/templates/plant.java.ftl

```diff
@@ -281,7 +282,15 @@ import net.minecraft.block.material.Material;
 					.tickRandomly()
 					</#if>
 					.doesNotBlockMovement()
-					.sound(SoundType.${data.soundOnStep})
+					<#if data.isCustomSoundType>
+						.sound(new ForgeSoundType(1.0f, 1.0f, new SoundEvent(new ResourceLocation("${data.breakSound}")),

```

Comment:
You need to use suppliers for the ForgeSoundType, add `() -> ` before each `new SoundEvent`